### PR TITLE
Initial parser for SARD database

### PR DESCRIPTION
**Pull request by dirkseidensticker**
Ready to be tested and if working, merging into master

### CODE REVIEW

File: R/get_sard.R [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        "Site" = "character",
+        "Country" = "character",
+        "Stratigraphic context" = "character",
+        "Archaeological Period" = "character",
+        "Archaeological Sub-chronology" = "character",
+        "DecdegS" = "character",
+        "DecdegE" = "character",
+        "Notes" = "character",
+        "refcode" = "character"
+      ),
```
Comment (nevrome) [inline R/get_sard.R:26]: `colClasses = "character"` is sufficient, as I recently learned.

File: R/get_sard.R [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      c14std = .data[["Uncertainty"]],
+      c13val = .data[["delta 13"]],
+      species = .data[["Species"]],
+      material = .data[["Material dated"]],
+      method = .data[["Dating technique"]],
+      site = .data[["Site"]],
+      country = .data[["Country"]],
+      feature = .data[["Stratigraphic context"]],
+      culture = .data[["Archaeological Period"]],
+      period = .data[["Archaeological Sub-chronology"]],
```
Comment (nevrome) [inline R/get_sard.R:41]: I think the culture and period columns should be switched. The definition doesn't fit perfectly here, but from a quick glance at the values it makes more sense to me to have it the other way round.

**Review by nevrome — COMMENTED** [review]
Cool!

File: R/get_sard.R (hunk 1 above)
Comment (dirkseidensticker) [reply R/get_sard.R:26]: This makes it way easier indeed 🥳

File: R/get_sard.R (hunk 2 above)
Comment (dirkseidensticker) [reply R/get_sard.R:41]: You are very much right 🎱

**Pull request merged**